jerry-core/jerry-api.h

```diff
@@ -21,6 +21,7 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <inttypes.h>
```

Comment:
We should not pollute the API header, especially as the macros are only needed by the implementation. This include has a better place in jrt-libc-includes.h .